app/views/templates/_related_ministers.html.haml

```diff
@@ -0,0 +1,13 @@
+- if related_ministers.present?
```

Comment:
Does passing in the local variable `object` automatically set a variable to the name of the template?